diagnostic.md

```diff
@@ -9,17 +9,25 @@ Assume all code executes in strict mode.
 
 Please list the JavaScript primitives and give one example of each.
 
+String: let variable = "string"
+number: let variable = 2
+null: let variable = ""
+undefined: let variable
+
+
```

Comment:
An empty string in javascript `''` is actually still a string. Null is only `null`. You have to set it to `null`.

So:

```
let jordan = ''
// jordan -> string
let jordan
// jordan -> undefined
let jordan = null
// jordan -> null
```